roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -13,6 +13,9 @@ storage:
     accesskey: {{ openshift.hosted.registry.storage.s3.accesskey }}
     secretkey: {{ openshift.hosted.registry.storage.s3.secretkey }}
     region: {{ openshift.hosted.registry.storage.s3.region }}
+{%   if 'regionendpoint' in openshift.hosted.registry.storage.s3 %}
+    regionendpoint: {{ openshift.hosted.registry.storage.s3.regionendpoint }}
+{%   endif -%}
```

Comment:
This should be:
```
{%   endif -%}
```
The '-' is throwing off formatting for the bucket key below by removing leading spaces.